src/components/Calculator.jsx

```diff
@@ -15,29 +16,32 @@ const Calculator = () => {
-        <button type="button" value="7" onClick={calc}>7</button>
-        <button type="button" value="8" onClick={calc}>8</button>
-        <button type="button" value="9" onClick={calc}>9</button>
-        <button type="button" className="operator" value="*" onClick={calc}>x</button>
-        <button type="button" value="4" onClick={calc}>4</button>
-        <button type="button" value="5" onClick={calc}>5</button>
-        <button type="button" value="6" onClick={calc}>6</button>
-        <button type="button" className="operator" value="-" onClick={calc}>-</button>
-        <button type="button" value="1" onClick={calc}>1</button>
-        <button type="button" value="2" onClick={calc}>2</button>
-        <button type="button" value="3" onClick={calc}>3</button>
-        <button type="button" className="operator" value="/" onClick={calc}>÷</button>
-        <button type="button" className="zero" value="0" onClick={calc}>0</button>
-        <button type="button" className="decimal" value="." onClick={calc}>.</button>
-        <button type="button" className="equal-sign" value="=" onClick={calc}>=</button>
+    <div className="temp">
+      <div className="calculator">
+        <input type="text" className="calculator-screen" value={state.next || state.total || 0} disabled />
+        <div className="calculator-keys">
+          <button type="button" className="all-clear" value="all-clear" onClick={calc}>AC</button>
+          <button type="button" className="plus-minus" value="plus-minus" onClick={calc}>+/-</button>
+          <button type="button" className="percent" value="percent" onClick={calc}>%</button>
+          <button type="button" className="operator" value="+" onClick={calc}>+</button>
+          <button type="button" value="7" onClick={calc}>7</button>
+          <button type="button" value="8" onClick={calc}>8</button>
+          <button type="button" value="9" onClick={calc}>9</button>
+          <button type="button" className="operator" value="*" onClick={calc}>x</button>
+          <button type="button" value="4" onClick={calc}>4</button>
+          <button type="button" value="5" onClick={calc}>5</button>
+          <button type="button" value="6" onClick={calc}>6</button>
+          <button type="button" className="operator" value="-" onClick={calc}>-</button>
+          <button type="button" value="1" onClick={calc}>1</button>
+          <button type="button" value="2" onClick={calc}>2</button>
+          <button type="button" value="3" onClick={calc}>3</button>
+          <button type="button" className="operator" value="/" onClick={calc}>÷</button>
+          <button type="button" className="zero" value="0" onClick={calc}>0</button>
+          <button type="button" className="decimal" value="." onClick={calc}>.</button>
+          <button type="button" className="equal-sign" value="=" onClick={calc}>=</button>
+        </div>
       </div>
+      <Quote />
```

Comment:
- [ ] Good job @fickryiman 😃, however, it could be better if  the `<Quote>` component  is imported into the main component the, the `App` component 👍 

````
function App() {
  return (
    <>
    <Calculator />
    <Quote />
    </>
  );
}
````

Reply:
move the <Quote /> component from the calculator component to the main component (App.js)

Reply:
Thank you ☺️